var/spack/repos/builtin/packages/tabix/package.py

```diff
@@ -0,0 +1,49 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Tabix(MakefilePackage):
+    """Generic indexer for TAB-delimited genome position files"""
+
+    homepage = "https://github.com/samtools/tabix"
+    url      = "https://github.com/samtools/tabix"
+
+    version('2013-12-16', git='https://github.com/samtools/tabix.git', commit='1ae158ac79b459f5feeed7490c67519b14ce9f35')
+
+    depends_on('perl', type=('build', 'run'))
+    depends_on('python', type=('build', 'run'))
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
+        install('tabix', prefix.bin)
+        install('bgzip', prefix.bin)
+        install('tabix.py', prefix.bin)
+        install('tabix.1', prefix.bin)
+        install('tabix.tex', prefix.bin)
```

Comment:
These two look like documentation. Maybe put them in a different directory? Manpages should go in `prefix.man.man1` or `prefix.share.man.man1`. I would put TeX docs in `prefix.share`.